src/backend/drive/settings.py

```diff
@@ -418,6 +418,28 @@ class Base(Configuration):
     FRONTEND_THEME = values.Value(
         None, environ_name="FRONTEND_THEME", environ_prefix=None
     )
+    FRONTEND_MORE_LINK = values.Value(
```

Comment:
You can probably group all these settings in one ? And build dynamically the feedback buttons based on this settings ? It's a little bit weird to ship code with `tchap` as name, it's specific to our use case